packages/safe-ds-lang/tests/language/runner/safe-ds-runner.test.ts

```diff
@@ -0,0 +1,56 @@
+import { describe, expect, it } from 'vitest';
+import { createSafeDsServicesWithBuiltins } from '../../../src/language/index.js';
+import { NodeFileSystem } from 'langium/node';
+import net from 'net';
+import { URI } from 'langium';
+
+const services = (await createSafeDsServicesWithBuiltins(NodeFileSystem)).SafeDs;
+const runner = services.runtime.Runner;
+
+describe('SafeDsRunner', async () => {
+    it('findFirstFreePort-occupied', async () => {
+        const server = net.createServer();
+        server.on('error', (err: any) => {
+            throw err;
+        });
+        const portNumber = 46821;
+        await new Promise<void>((resolve, _reject) => {
+            server.listen(portNumber, '127.0.0.1', () => {
+                resolve();
+            });
+        });
+        const foundPort = await runner.findFirstFreePort(portNumber);
+        server.close();
+        expect(foundPort).toStrictEqual(portNumber + 1);
+    });
+    it('findFirstFreePort-available', async () => {
+        const portNumber = 46825;
+        const foundPort = await runner.findFirstFreePort(portNumber);
+        expect(foundPort).toStrictEqual(portNumber);
+    });
```

Comment:
Wrap the test cases (`it`) in test suites (`describe`) for the individual functions inside the top-level `describe(`SafeDsRunner`)`.